src/main/scala/org/broadinstitute/hail/driver/Main.scala

```diff
@@ -38,6 +38,9 @@ object Main {
     @Args4jOption(required = false, name = "--master", usage = "Set Spark master (default: system default or local[*])")
     var master: String = _
 
+    @Args4jOption(name = "-b", aliases = Array("--blocksize"), usage = "Minimum size of file system splits")
+    var blockSize: Int = 128
```

Comment:
Sorry to come late to this, but it might be better to have a more explicit name for the flag, like `--min-split-size`. "Block" is a pretty overloaded term.

Also, it's not clear it's in MB.